### ISSUE TITLE
Stdlib documentation problem

### ISSUE THREAD

**Issue by ttamttam** [ocaml/ocaml#8736]
Hello,

in https://caml.inria.fr/pub/distrib/ocaml-4.08, the Stdlib part of the documentation is weired.

Extracted from https://caml.inria.fr/pub/distrib/ocaml-4.08/ocaml-4.08-refman.txt: the Module Stdlib section seems to be included twice:
```
- Chapter 23  The core library 
        
         - 23.1  Built-in types and predefined exceptions 
         - 23.2  Module Stdlib : The OCaml Standard library. 
         - 23.3  Module Stdlib : The OCaml Standard library. 
```

And the standard modules documentations are missing:
```
Standard library modules 
=========================
   
<<
  module Arg : >>
   
  Arg
<<
  module Array : >>
   
  Array
<<
  module ArrayLabels : >>
   
  ArrayLabels
<<
  module Bigarray : >>
   
  Bigarray
```

Best regards

**Comment by ttamttam** [comment]
https://caml.inria.fr/pub/distrib/ocaml-4.08 seems to be a temporary distribution.
The manual in http://caml.inria.fr/pub/docs/manual-ocaml-4.08/core.html is OK.
Closing, and sorry for the noise.

**Comment by alan-j-hu** [comment]
In the 4.08 version of the manual, the list of standard library modules is accessible here: https://caml.inria.fr/pub/docs/manual-ocaml-4.08/stdlib.html

It is listed as chapter 26.

However, the main page (https://caml.inria.fr/pub/docs/manual-ocaml-4.08/index.html) does not link to that page and it lists the num library documentation as chapter 26.

I noticed that the Int module documentation link leads to a 404: https://caml.inria.fr/pub/docs/manual-ocaml-4.08/libref/Int.

**Comment by pmetzger** [comment]
I suspect a pull request with a fix might be noticed faster than a normal bug report...

**Comment by Octachron** [comment]
This was fixed with #8767 .
